### PR TITLE
Support Python 3.10

### PR DESCRIPTION
**Pull request by nguyenv**

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+
 # TileDB-Py 0.11.2 Release Notes
+
 ## TileDB Embedded updates:
 * TileDB-Py 0.11.2 includes TileDB Embedded [TileDB 2.5.2](https://github.com/TileDB-Inc/TileDB/releases/tag/2.5.2)
 
 # TileDB-Py 0.11.1 Release Notes
+
 ## Bug fixes
 * Support dict parameter for 'config' argument to VFS constructor [#805](https://github.com/TileDB-Inc/TileDB-Py/pull/805)
```
Comment (ihnorton) [inline HISTORY.md:14]: Would you mind to move this to the correct section (0.11.2)?

File: misc/requirements_wheel.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # numpy pinning for ABI forward-compatibility
 numpy==1.16.5 ; python_version < "3.8" and platform_machine !='aarch64'
 numpy==1.17.* ; python_version == "3.8" and platform_machine !='aarch64'
-numpy==1.19.4 ; python_version >= "3.9" and platform_machine !='aarch64'
+numpy==1.19.4 ; python_version == "3.9" and platform_machine !='aarch64'
 
 # NOTE: oldest-supported-numpy (1.19.2) had forward ABI compat problems
-numpy==1.20.* ; platform_machine=='aarch64'
+numpy==1.21.*
```
Comment (ihnorton) [inline misc/requirements_wheel.txt:7]: I think the change should be:
```
numpy==1.20.* ; python_version < "3.10" and platform_machine=='aarch64'
numpy == 1.21.* ; python_version >= "3.10"
```

- keeps the 1.20 pin for python 3.9 + aarch64
- pin 1.21 for >=3.10 unless/until we have to manually adjust for another release

**Review by ihnorton — APPROVED** [review]
Thanks!!

File: HISTORY.md (hunk 1 above)
Comment (nguyenv) [reply HISTORY.md:14]: Do you mean move "Support dict parameter..." into 0.11.2 or something else?

Comment (ihnorton) [reply HISTORY.md:14]: > Do you mean move "Support dict parameter..." into 0.11.2

yes

**Pull request merged**